src/backendLogic.js

```diff
@@ -38,26 +45,102 @@ app.get('/', async (req, res) => {
 app.post('/check_login_info', async (req, res) => {
     const fileContent = await getFileContent(`${rootPath}/data/user_list.json`);
     const userList = JSON.parse(fileContent).users;
-    let isValidUser = false;
-    for (i = 0; i < userList.length; i++) {
-        if (userList[i].username === req.body.username) {
-            isValidUser = true;
-            if (userList[i].password === req.body.password) {
-                res.send(JSON.stringify("Login Success"));
-            } else {
-                res.statusCode = 401;
-                res.send(JSON.stringify("Wrong Password"));
-            }
-        }
-    }
-    if (!isValidUser) {
-        res.statusCode = 400;
-        res.send(JSON.stringify("Invalid User"));
+    const username = req.body.username;
+    const password = req.body.password;
+
+    switch (await checkLoginInfo(username, password)) {
+        case LOGIN_SUCCESS:
+            res.send(JSON.stringify(LOGIN_SUCCESS));
+            break;
+        case WRONG_PASSWORD:
+            res.statusCode = 401;
```

Comment:
Let's use this with http code name, not just numbers.
https://www.npmjs.com/package/http-status-codes

Reply:
okie